index.js

```diff
@@ -1,25 +1,31 @@
 var tcp = require('../../tcp');
 var instance_skel = require('../../instance_skel');
+var fs = require('fs');
 
 var instance_api  = require('./internalAPI');
 var actions       = require('./actions');
 var feedback      = require('./feedback');
 var presets       = require('./presets');
 var variables     = require('./variables');
+const { isNull }  = require('util');
+const { getOutput } = require('./internalAPI');
```

Comment:
@BaseballJim this shouldn't be necessary because the entire palette of api (incl. @ line 5) functions are merged into `this` at line 50.